### PR TITLE
[PF-814] add serviceAccountTokenCreator

### PR DESCRIPTION
**Pull request by jaycarlton**
According to this error message, I need to have the `serviceAccountTokenCreator` role on the main control plane service account:
```
com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException:
FAILED_PRECONDITION: P4 service account needs iam.serviceAccounts.getAccessToken permission.
Running the following command may resolve this error:
gcloud iam service-accounts add-iam-policy-binding workspace-wsmtest@terra-kernel-k8s.iam.gserviceaccount.com
  --member='serviceAccount:service-157747328908@gcp-sa-bigquerydatatransfer.iam.gserviceaccount.com'
  --role='roles/iam.serviceAccountTokenCreator'
```

### CODE REVIEW

**Review by jmczerk — APPROVED** [review]

File: terra-env/main.tf
Comment (ddietterich) [inline terra-env/main.tf]: I'm not conversant in TF. Why does this have your name in it? Is it temporary? Does it need versioning?

**Review by ddietterich — COMMENTED** [review]
Some questions
p.s. this repo only needs one approval, so you can merge if you are confident of what you're doing. This is not an area of code I know.

File: terra-env/main.tf
Comment (jaycarlton) [reply terra-env/main.tf]: It's temporary. It will be replaced, after approval, by a new tag value.

**Review by ddietterich — COMMENTED** [review]
LGTM

**Review by ddietterich — APPROVED** [review]
LGTM

**Pull request merged**
